forum123.py

```diff
@@ -0,0 +1,4 @@
+"""An entry point to forum123."""
```

Comment:
Seems like we don't need this file, since flask application can be run using `FLASK_APP=src flask run`